lib/errors.js

```diff
@@ -88,6 +88,25 @@ class ValidationError extends BaseError {
 exports.ValidationError = ValidationError;
 
 /**
+ * Thrown when attempting to update a stale model instance
+ * @extends BaseError
+ * @constructor
+ */
+class OptimisticLockError extends BaseError {
+  constructor(options) {
+    options = options || {};
+    options.message = options.message || 'Attempting to update a stale model instance: ' + options.modelName;
+    super(options);
+    this.name = 'OptimisticLockError';
```

Comment:
Should be `SequelizeOptimisticLockError` to be consistent with the other error names